.mci.yml

```diff
@@ -601,6 +618,7 @@ axes:
                 cmake_flags: *linux_cmake_flags
                 build_type: "Debug"
                 tar_options: *linux_tar_options
+                code_coverage_cmake_flags: *code_coverage_cmake_flags
```

Comment:
Awesome! Looking at the runtime difference, it seems like tests only take one more minute with code coverage enabled. That seems like a worthwhile trade.

We'd likely get even more accurate coverage if we run the replica set tests with code coverage as well (e.g. transactions tests only run against a replica set). Currently there's only one replica set task that's run on the release variant.

If we add `compile_and_test_with_shared_libs_replica_set` to the "integration" matrix below, that may do the trick. Though there may be some unexpected failures / additional setup.

No need to do this now if it's non-trivial. This is an overall improvement as-is.

Reply:
Ah, good call. I was able to test against a replica set for Linux only. Windows fails so I left out of this commit.